### PR TITLE
Prettier error pages

### PR DESCRIPTION
**Pull request by promatik**
Quick question before merging;
This PR adds a translation;

image

**"Take me Home"** button.

What should we do about it?
1) translate in the theme?
2) translate in crud?
3) remove the button so we don't need to add translations?

I think we should make the theme translatable, because there are other string there, that are currently not translated 😬
And others my come up, like the in the https://github.com/Laravel-Backpack/theme-tabler/pull/120

---
Depends on https://github.com/Laravel-Backpack/CRUD/pull/5276

### CODE REVIEW

File: resources/views/errors/layout.blade.php [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+@section('content')
+<div class="empty">
+  <div class="empty-header">{{ $error_number }}</div>
+    <p class="empty-title">@yield('title')</p>
+    <p class="empty-subtitle text-secondary">@yield('description')</p>
+    <div class="empty-action">
+      <a href="./." class="btn btn-primary">
+        <svg xmlns="http://www.w3.org/2000/svg" class="icon" width="24" height="24" viewBox="0 0 24 24" stroke-width="2" stroke="currentColor" fill="none" stroke-linecap="round" stroke-linejoin="round"><path stroke="none" d="M0 0h24v24H0z" fill="none"></path><path d="M5 12l14 0"></path><path d="M5 12l6 6"></path><path d="M5 12l6 -6"></path></svg>
+        {{ trans('Take me home') }}
```
Comment (tabacitu) [inline resources/views/errors/layout.blade.php:11]: 🚩 We cannot use translations from the general namespace. If we do use translations, it should be from THIS PACKAGE.

**Comment by tabacitu** [comment]
In this case, perhaps we can avoid the translation, by having buttons that show icons for BACK and HOME?

**Comment by tabacitu** [comment]
Otherwise I'm ok with adding these two translation strings to CRUD. They can be used in the default error views. Then themes use them too when they customize the error views. Makes sense right?

**Comment by promatik** [comment]
Translations added here; https://github.com/Laravel-Backpack/theme-tabler/pull/136

File: resources/views/errors/layout.blade.php (hunk 1 above)
Comment (promatik) [reply resources/views/errors/layout.blade.php:11]: Solved with https://github.com/Laravel-Backpack/theme-tabler/pull/136

**Review by pxpm — CHANGES_REQUESTED** [review]
Hey @promatik this needs to be updated to account for the Horizontal Overlap 

image

I've added sometime ago a card wrapper on the default error pages on CRUD (default ui) https://github.com/Laravel-Backpack/CRUD/pull/5239

I also notice that the error text is way smaller than what it was before:
image

Cheers

**Pull request merged**